tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "checkJs": false,
```

Comment:
`checkJs` sounds like something we'd prefer to be enabled. Is there a good reason to disable it?